.github/workflows/test.yml

```diff
@@ -27,6 +27,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - run: swift test --enable-test-discovery --sanitize=thread
-      env:
-        REDIS_HOSTNAME: redis
+    - run: apt update && apt install libssl-dev zlib1g-dev -y && swift test --enable-test-discovery --sanitize=thread
+    env:
+      REDIS_HOSTNAME: redis
+  # xenial:
```

Comment:
all the commented out code should be deleted but I think we should stick to the vapor images